### DISCUSSION TITLE
Cobo Vault doesn't see other signatures in partially signed transaction

### DISCUSSION THREAD

**Discussion by wrengulf** [cryptoadvance/specter-desktop#801] [Hardware Wallets]
Not sure if this is a specter or Cobo issue but if I scan a QR code from Specter that is partially signed by other devices, the Cobo "loses" those signatures and sees the transaction as "unsigned". If I sign with Cobo first, I can then get the necessary other device signatures. Anyone else had this issue ?

**Comment by ben-kaufman** [comment]
Hi, I don't think the Cobo needs to "see" the signatures. Specter is coordinating the signing and aggregates the independent signatures, so the individual hardware wallets don't need any information about other signatures...

**Comment by wrengulf** [comment]
Thanks Ben, what I mean is that if I sign first with another device (trezor, Coldcard etc) and then scan that QR into Cobo Vault from Specter, when I sign on Cobo and then scan the QR back into Specter... Specter says the transaction only has one signature (from Cobo) and has lost the signature from the other device. I'm guessing that my problem is unique to me but I can't figure out where the issue might lie.

**Reply by ben-kaufman** [reply]
Just checked, works fine for me. Where are you scanning it from back into Specter? (Need to know exactly which button you're clicking on to open the camera in specter)
